src/Twig/FileExtension.php

```diff
@@ -0,0 +1,101 @@
+<?php
+declare(strict_types=1);
+
+namespace Chialab\FrontendKit\Twig;
+
+use Cake\I18n\Number;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+
+/**
+ * File extension for Twig.
+ */
+class FileExtension extends AbstractExtension
+{
+    /**
+     * @inheritDoc
+     */
+    public function getFilters()
+    {
+        return [
+            new TwigFilter('readable_size', [$this, 'readableSize']),
+            new TwigFilter('mime_type', [$this, 'mimeType']),
+        ];
+    }
+
+    /**
+     * Format bytes size.
+     *
+     * @param int|string $size Byte size.
+     * @return string
+     */
+    public function readableSize($size): string
+    {
+        return Number::toReadableSize($size);
+    }
+
+    /**
+     * Get the generic type of the mime.
+     *
+     * @param string $mime The mime type string.
+     * @return string
+     */
+    public function mimeType(string $mime): string
```

Comment:
I find the name of this filter really counter-intuitive.

This filter converts a standard media type (only formerly known as MIME type) into a completely arbitrary and non-standard "category". I'd use a more self-explanatory name. Maybe something like `mime2category`?